experiments/integrated/performance/conservation/ozark_conservation_setup.jl

```diff
@@ -192,8 +192,15 @@ land = SoilCanopyModel{FT}(;
     canopy_model_args = canopy_model_args,
 )
 exp_tendency! = make_exp_tendency(land)
```

Comment:
Interestingly, this file was already using ARS111 and an IMEXAlgorithm (lines 18-32 of this file)

Reply:
oh! that's totally it. We were using explicit euler before, but the only option for that is IMEX ars111 (with imp_tendency! doing nothing). there is no pure explicit euler option in ClimaTimestppers. this matters just because the expected energy and water changes we compute are computed assuming explicit euler timesteps. 

like, if Var(t+dt)= Var(t) + function(fluxes(t)) * dt, we can check conservation by comparing Var(t+dt) - Var(t) = function(fluxes(t)) * dt (but over many steps - and just expect roundoff error)

but if the timestepper changes, this is not the case. if it was implicit, we could do
Var(t+dt) - Var(t) = function(fluxes(t+dt)) * dt

basically im not really sure how to compute the "truth" for higher order imex schemes.

Reply:
maybe we can ask Dennis. We mostly want to verify that d(energy in land) = energy in - energy out over the whole sim. 

I think the important thing is that this is not growing. so I am inclined to think this is OK, we just are computing what we want

Reply:
Ohh I see! Would it be possible to use the equation you gave for the explicit case for explicitly-stepped variables, and the implicit one for implicitly-stepped variables? Or does that not make sense because we have to consider both variables and fluxes?

Reply:
im not sure! maybe we can discuss more tomorrow? Im not worried about this really for this PR though... we maybe need to rethink that conversation test in general

Reply:
happy to discuss tomorrow! I opened an issue for this too so we can keep track of it https://github.com/CliMA/ClimaLand.jl/issues/662